src/client/sandbox/node/live-node-list/html-collection-wrapper.js

```diff
@@ -0,0 +1,144 @@
+import DOMMutationTracker from './dom-mutation-tracker';
+import nativeMethods from '../../native-methods';
+import { isShadowUIElement } from '../../../utils/dom';
+
+// NOTE: Tags from https://www.w3schools.com/tags/att_name.asp
+const ELEMENTS_WITH_NAME_ATTRIBUTE = ['button', 'fieldset', 'form', 'iframe',
+    'input', 'map', 'meta', 'object', 'output', 'param', 'select', 'textarea'];
+const DEFAULT_SHADOW_GETTERS_SHIFT = 10;
+let shadowGettersCount             = 0;
+
+export default function HTMLCollectionWrapper (htmlCollection, tagName) {
+    tagName = tagName.toLowerCase();
+
+    nativeMethods.objectDefineProperties.call(Object, this, {
+        htmlCollection:         { value: htmlCollection },
```

Comment:
`html` suffix is an unneeded context. We is already in `HTMLCollectionWrapper` context.
So, remove `html` suffix from variable and function names (`htmlCollection`. `refreshHmtlCollection` and etc.)